R/create_figures_doc.R

```diff
@@ -120,8 +125,12 @@ if (file.exists(file.path(rda_dir, 'spawning.biomass_figure.rda'))){\n
       ),
       "\n"
     )
+    } else {
+      message("Spawning biomass time series figure not created.")
+    }
 
     # B figure
+    if (any(grepl("biomass_figure.rda", list.files(file.path(rda_dir, "rda_files"))))) {
```

Comment:
It's creating chunks to plot the biomass figure even if I don't have the biomass rda. Guessing it's related to the grepl fxn looking for biomass and finding spawning biomass?